proto_converter.go

```diff
@@ -113,13 +113,13 @@ func (converter *protoConverter) toField(key string, value interface{}) *cpb.Key
 	case reflect.Bool:
 		field.Value = &cpb.KeyValue_BoolValue{BoolValue: reflectedValue.Bool()}
 	default:
-		field.Value = &cpb.KeyValue_StringValue{StringValue: fmt.Sprint(reflectedValue)}
-		maybeLogInfof(
-			"value: %v, %T, is an unsupported type, and has been converted to string",
-			converter.verbose,
-			reflectedValue,
-			reflectedValue,
-		)
+		var s string
```

Comment:
due to the funky branch merging going on, this is a change I pulled in from a fix Josh made recently. https://github.com/lightstep/lightstep-tracer-go/pull/114

Reply:
:git_confusion: